specification/azsadmin/resource-manager/compute/Microsoft.Compute.Admin/preview/2018-07-30-preview/Disks.json

```diff
@@ -0,0 +1,319 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "2018-07-30-preview",
+    "title": "ComputeDiskAdminManagementClient",
+    "description": "The Admin Compute Disk Management Client."
+  },
+  "host": "adminmanagement.local.azurestack.external",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute.Admin/locations/{location}/disks": {
+      "get": {
+        "x-ms-examples": {
+          "Returns a list of disks.": {
+            "$ref": "./examples/Disks/List.json"
+          }
+        },
+        "tags": [
+          "Disks"
+        ],
+        "operationId": "Disks_List",
+        "description": "Returns a list of disks.",
+        "parameters": [
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../2015-12-01-preview/Compute.json#/parameters/LocationParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/TenantSubscriptionIdParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStateParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/SourceSharePathParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskCountParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/DiskStartParameter"
+          },
+          {
+            "$ref": "Disks.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK -- The list of disks has been returned.",
+            "schema": {
+              "$ref": "Disks.json#/definitions/DiskList"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
```

Comment:
Why? RPC requires all collections to be "page-able". You can set property to null on the server side, but from client side, they should always assume that you may start paging one day.

Reply:
The rest API itself doesn't support the nextLink right now. So mark it as null and it will retrieve the output out as List.